README.md

```diff
@@ -0,0 +1,138 @@
+
+The cofact distribution includes a script called `run_all` that will run cofact on each Fermat number from $F_0$ through $F_{29}$ using the best method for that number, with "best" meaning reasonably fast. Before running the script, download proof files for $F_{17}$ through $F_{29}$ into the same directory as cofact.
+
+## Command line options
+The following command line options are supported by cofact (main branch):
+Command line option | Function
+--------------------|------------------------------
+-cpr _file_         | Read the Suyama A residue from the mprime proof file and compare it to the A residue calculated by cofact (mode 2)
+-d                  | Print debug information
+-h                  | Print this help and exit
+-p _iter_           | Print progress every iter iterations, instead of the default of every 10% of total iterations for longer runs
+-sep                | Print a separator at the end of the run to better see multiple run's output in a single output file
+-t _threads_        | Specifies the number of threads to use in the gwnum library. Defaults to 1.
+-upr _file_         | Read the Suyama A residue from the mprime proof file and use it to complete the Suyama test (mode 3)
+-v                  | Print more verbose information
+
+## Authors
+Gary B. Gostin (gary641), versions 0.2 to 0.8.2 (the original, `main` branch)
+
+Catherine X. Cowie (xanthe-cat), versions 0.6 to 0.9 (the `cxc` branch)
+
+
+## Copyright
+This program is © 2023–2024 Gostin and Cowie under the Creative Commons Zero (CC-0) licence. We will not be responsible for whether you find this work useful, or whether it opens a portal to another hellish dystopian dimension where your computer used to be.
+
+The gwnum library is © 2002–24 Mersenne Research, Inc, used with permission. All rights reserved.
+
+The GMP library is © 1991, 1993–2016, 2018–2024 Free Software Foundation, Inc.
+
+## Mathematics
+A Fermat number is an integer of the form $2^{2^n}+1$, where $n \ge 0$. The five smallest Fermat numbers, $F_0$ through $F_4$, are prime. $F_5$ through $F_{11}$ have been fully factored. $F_{20}$ and $F_{24}$ have been proven composite but not factored, while many other Fermat numbers have known factors.
+
+A Fermat cofactor is the integer $F_n / P$, where $P$ is the product of all known factors of $F_n$.
+
+### Pépin's test
+Pépin's test is a definitive primality test that can be used to determine if a Fermat number is prime. If $F_n$ is the $n^{th}$ Fermat number, then $F_n$ is prime if and only if $3^{(F_n - 1)/2} \equiv -1$ (mod $F$). The term $3^{(F_n - 1)/2}$ (mod $F$) may be calculated by $2^n-1$ squarings of the base $3$, modulo $F_n$. Pépin’s test is also a definitive primality test when performed with bases other than 3, such as 5, 6, and so on.
```

Comment:
This paragraph uses Pépin with the acute accent, so if you want to copy / paste this accented version to the rest of the 20 or so uses of Pepin elsewhere in the Readme, you would make my inner pedant very happy!

Reply:
Good suggestions. Fixed.